ConsoleAppForTesting/MultiThreadingHomeWork.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace MultiThreading
+{
+    using System;
+    using System.Threading;
+    using ConsoleAppForTesting;
+
+    public static class MultiThreadingHomeWork
+    {
+        static void Main(string[] args)
+        {
+
+            //var myMutex = new Mutex();
+            //var xLocker = 0;
+            //Object locker = new {};
+            //Console.WriteLine("Thread # {0} now start working, locker = {1} ", Thread.CurrentThread.ManagedThreadId, xLocker);
+            //Thread myThread;
```

Comment:
If you need the commented code just move it to separate method, do not leave commented code.